app/controllers/second_level_browse_page_controller.rb

```diff
@@ -22,8 +22,11 @@ def show
   def show_html
     taxon_resolver = TaxonRedirectResolver.new(
       request,
-      is_page_in_ab_test: lambda { params[:top_level_slug] == "education" },
-      map_to_taxon: lambda { redirects["education"][params[:second_level_slug]] }
+      is_page_in_ab_test: lambda {
+        !redirects[params[:top_level_slug]].nil? &&
```

Comment:
Could this be better expressed?
eg `redirects.dig(params[:top_level_slug], [params[:second_level_slug])`